react/features/conference-messages/reducer.js

```diff
@@ -0,0 +1,44 @@
+// @flow
+
+import { ReducerRegistry } from '../base/redux';
+
+import {
+    HIDE_CONFERENCE_MESSAGE,
+    SET_CONFERENCE_MESSAGE
+} from './actionTypes';
+
+/**
+ * The name of the Redux subtree that this feature stores its state in.
+ */
+const STATE_NAME = 'features/conference-messages';
+
+/**
+ * Reduces the Redux actions of the feature features/etherpad.
+ */
+ReducerRegistry.register(STATE_NAME, (state = {}, action) => {
+    switch (action.type) {
+
+    case HIDE_CONFERENCE_MESSAGE: {
+        const { messageKey: keyToHide } = action;
+        const { messageKey } = state;
+
+        if (!keyToHide || messageKey === keyToHide) {
+            return {
+                ...state,
+                messageKey: undefined
+            };
+        }
+
+        return state;
+    }
+
+    case SET_CONFERENCE_MESSAGE:
+        return {
+            ...state,
+            messageKey: action.messageKey
+        };
+
+    default:
+        return state;
+    }
+});
```

Comment:
I remember now (great timing, right?) I was also thinking of asking to get the web recording messages redesigned. Maybe on web it would be good to leverage the notifications feature for recording messages. ConferenceMessages sound a lot like Notifications to me. I never asked...

Reply:
That is a very good point, and I'm working on it right now. The feature "notifications" will be unified over web and mobile and both platform will use them for recording messages